Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM centos:centos6 AS builder
+
```

Comment:
@tajayi Now there would be two dockerfiles dealing with building fastroute.  

As much as possible, each module should be an independent of the others. There should be a tree of dependencies between the modules, with no circular dependencies. 

I would expect the Dockerfile for this module to be a concern of this module and that creating the overall docker image for all the modules in OpenROAD should be a separate concern for alpha-release, so that alpha-release should use the Dockerfile in this image.

https://github.com/The-OpenROAD-Project/alpha-release/blob/master/build/docker/fastroute/Dockerfile

Every module in OpenROAD should have a module such as this one. Fixing this problem for fastroute should set a pattern to be followed for the other modules.

Reply:
Just to give some context, we are working on `develop` branch of the alpha release. https://github.com/The-OpenROAD-Project/alpha-release/tree/develop

The goal is to **not** have Dockerfiles separated from the repositories. So, ultimately, the `build` folder inside the alpha-release will be deleted and we will have one source of truth for building any component in the OpenROAD project - its accompanying Dockerfile. Build instructions for the alpha-release will be a the instructions used to build and export the individual components. We also want Dockerfiles to be in their own repositories to enable Jenkins pipelines for automatic integration and build/test.